README.md

```diff
@@ -1517,6 +1517,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
    * [codesandbox.io](https://codesandbox.io/) — Online Playground for React, Vue, Angular, Preact and more.
    * [Components.studio](https://webcomponents.dev/) - Code components in isolation, visualize them in stories, test them and publish them on npm.
    * [Eclipse Che](https://www.eclipse.org/che/) - Web based and Kubernetes-Native IDE for Developer Teams with multi-language support. Open Source and community driven. A online instance hosted by Red Hat is available at [workspaces.openshift.com](https://workspaces.openshift.com/).
+   * [DevZero](https://devzero.io/) — Cloud dev environments for various frameworks and language. Connected to local IDEs like VS Code and Jetbrains. Unlimited usage in free tier.
```

Comment:
Can you link to the free tier info please? Pricing lists a 30 day free trial only